pandas/core/base.py

```diff
@@ -1185,6 +1185,10 @@ def drop_duplicates(self, keep='first', inplace=False):
               occurrence.
             - False : Mark all duplicates as ``True``.
         take_last : deprecated
+        shortcircuit : boolean, False
+            return True if ANY value is duplicated, False otherwise
```

Comment:
this shortcircuits by design. ``is_unique`` will evaluate the entire space. Its almost an internal option, but would need to duplicate a lot of code; this is simpler.